docs/01-app/05-api-reference/03-file-conventions/parallel-routes.mdx

```diff
@@ -276,9 +276,9 @@ export default function Default() {
 }
 ```
 
-Inside your `@auth` slot, intercept the `/login` route by updating the `/(.)login` folder. Import the `<Modal>` component and its children into the `/(.)login/page.tsx` file:
+Inside your `@auth` slot, intercept the `/login` route by importing the `<Modal>` component and its children into the `/login/page.tsx` file, and updating the folder name to `/(..)login/page.tsx`.
```

Comment:
I think here it has be `(.)` though, because it is we want to intercept navigation to `/login` from the slot`/app/@auth`, which is a root level slot, so we need to intercept at the same level